client/src/components/NavBar/NavBar.tsx

```diff
@@ -3,41 +3,60 @@ import { AppBar, Toolbar, Button, IconButton, Link, Box } from '@material-ui/cor
 import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
 import { Link as routerLink } from 'react-router-dom';
 import NotificationsMenu from './NotificationsMenu/NotificationsMenu';
+import AuthHeader from '../../components/AuthHeader/AuthHeader';
+
 export default function NavBar(): JSX.Element {
   const { loggedInUser } = useAuth();
 
   return (
     <Box flexGrow={1}>
-      {loggedInUser && (
-        <AppBar color="secondary" position="static">
-          <Box padding={1}>
-            <Toolbar>
-              <Box flexGrow={1} display={{ xs: 'none', sm: 'block' }}>
-                <Link component={routerLink} to="/dashboard">
-                  <img
-                    src="https://res.cloudinary.com/dalisapxa/image/upload/v1630805884/DEV/logo_jizgel.png"
-                    alt="loving sitter"
-                  />
-                </Link>
-              </Box>
+      <AppBar color="secondary" position="static">
+        <Box padding={1}>
+          <Toolbar>
+            <Box flexGrow={1} display={{ xs: 'none', sm: 'block' }}>
+              <Link component={routerLink} to="/dashboard">
+                <img
+                  src="https://res.cloudinary.com/dalisapxa/image/upload/v1630805884/DEV/logo_jizgel.png"
+                  alt="loving sitter"
+                />
+              </Link>
+            </Box>
 
-              <NotificationsMenu />
-              <Box padding={1} marginLeft={1}>
-                <Button>My jobs</Button>
-              </Box>
-              <Box padding={1} marginLeft={1}>
-                <Button>Messages</Button>
-              </Box>
+            {loggedInUser && (
+              <>
+                <Box padding={1} marginLeft={1}>
+                  <Link component={routerLink} to="/dashboard">
+                    <Button>Dashboard</Button>
+                  </Link>
+                </Box>
+                <NotificationsMenu />
+                <Box padding={1} marginLeft={1}>
+                  <Button>My jobs</Button>
+                </Box>
+                <Box padding={1} marginLeft={1}>
+                  <Button>Messages</Button>
+                </Box>
+                <Box padding={1} marginLeft={1}>
+                  <Link component={routerLink} to="/my-sitters">
+                    <Button>My Sitters</Button>
+                  </Link>
+                </Box>
 
-              <Box padding={1} marginLeft={1}>
-                <IconButton component={routerLink} to="/settings/profile">
-                  <AvatarDisplay loggedIn user={loggedInUser} />
-                </IconButton>
-              </Box>
-            </Toolbar>
-          </Box>
-        </AppBar>
-      )}
+                <Box padding={1} marginLeft={1}>
+                  <IconButton component={routerLink} to="/settings">
+                    <AvatarDisplay loggedIn user={loggedInUser} />
+                  </IconButton>
+                </Box>
+              </>
```

Comment:
Is this block properly indented?

Reply:
I think so, prettier is not displaying problems